docs/source/reference/label_format/Box2D.rst

```diff
@@ -104,6 +104,52 @@ see :ref:`reference/label_format/CommonSubcatalogProperties:common category info
 :ref:`reference/label_format/CommonSubcatalogProperties:attributes information` and
 :ref:`reference/label_format/CommonSubcatalogProperties:tracking information` for details.
 
+
+The Box2D catalog is typically stored in a json file as follows::
+
+    {
+        "BOX2D": {                                        <object>*
+            "description":                                <string>! -- Catalog description, (default: "")
+            "isTracking":                                <boolean>! -- Whether this label in the dataset support for
+                                                                       tracking information, (default: false).
+            "categoryDelimiter":                          <string>  -- The delimiter used to split the parent-child
+                                                                       relationship between categories. The recommended
+                                                                       separator is ".". The default is that there are
+                                                                       no categories with parent-child relationships.
+            "categories": [                                <array>  -- Category list, which contains all category
+                                                                       information.
+                {
+                    "name":                               <string>* -- Category name
+                    "description":                        <string>! -- Category description, (default: "")
+                },
+                ...
+                ...
+            ],
+            "attributes": [                                <array>  -- Attribute list, which contains all attribute
+                                                                       information.
+                {
+                    "name":                               <string>* -- Attribute name
+                    "enum": [...],                         <array>  -- All possible options for the attribute.
+                    "type":                      <string or array>  -- Type of attribute, such as "boolean", "integer",
```

Comment:
`such as` or `all possible values`?
And address that `type` is not required when `enum` is provided.